app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -15,6 +15,9 @@
 
 import javax.transaction.Transactional;
 
+/**
+ * 회원을 찾고, 등록하고, 정보를 수정하고, 삭제합니다.
```

Comment:
4가지 기능 중 하나라도 삭제되거나 새로운 기능이 추가되면 이 주석도 수정하게 되겠죠.
```suggestion
 * 회원 관리 기능을 제공합니다.
```